howtos/introduction-of-a-new-tc-member.md

```diff
@@ -0,0 +1,12 @@
+# Introduction of a new TC member
+
+Checklist for introducing a new Technical Committee member
+
+- Become a owner of Eiffel Community GitHub organization

```

Comment:
**an** owner

Same thing in a few other bullet items below.

Reply:
Fixed in 47fd43f